protocols/mdns/src/behaviour.rs

```diff
@@ -150,7 +179,8 @@ impl Mdns {
             events: Default::default(),
             query_interval: config.query_interval,
```

Comment:
This needs to have the jitter as well, otherwise reset_timer and fire_timer remove the jitter again.

Reply:
Nice catch, yes, that won't work